src/shared/port-provider.js

```diff
@@ -162,12 +162,12 @@ export class PortProvider {
     }
 
     if (channel === 'guest-host' && message.frame1 === 'guest') {
-      this._emitter.emit('hostPortRequest', message.frame1, port2);
+      this._emitter.emit('hostFrameRequest', message.frame1, port2);
     }
   }
 
   /**
-   * @param {'hostPortRequest'} eventName
+   * @param {'hostFrameRequest'} eventName
```

Comment:
The important point for readers to know is that this event signals that a frame (currently the sidebar, but maybe other frames in future) has connected and the host may want to interact with it in some way. How about `frameConnected`?

Reply:
👍🏼